release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -667,6 +667,8 @@ This script removes unauthenticated subjects from the following cluster role bin
 +
 You can attempt to resolve the issue with the steps in this link:https://kb.vmware.com/s/article/2002779[VMware KBase article]. For more information, see the Red Hat Knowledgebase solution link:https://access.redhat.com/solutions/5785341[[UPI vSphere\] Node scale-up doesn't work as expected]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1918383[*BZ#1918383*])
 
+* The `oc annotate` command does not work for annotations that contain LDAP group names. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
```

Comment:
The `oc annotate` command does not work for LDAP group names containing `=` (equal sign) because the command uses it as a delimiter between annotation name and value.

Reply:
Thanks @soltysh I updated if you can take another look